personhood/service.go

```diff
@@ -265,26 +530,50 @@ func (s *Service) TopupMessage(tm *TopupMessage) (*StringReply, error) {
 	return &StringReply{}, nil
 }
 
+// TestStore allows easier testing of the mobile apps by giving an endpoint
+// where current testing data can be stored.
```

Comment:
It must have a better way to do that. It sounds really wrong to have to maintain a test endpoint for tests not even in the repo.

Reply:
I can maintain my own service for that...